.travis.yml

```diff
@@ -3,10 +3,9 @@ language: node_js
 node_js:
   # we recommend testing addons with the same minimum supported node version as Ember CLI
   # so that your addon works for all apps
-  - "6"
+  - "10"
```

Comment:
Can you do this (along with the package.json engines update) in a stand alone PR? Specifically to call out the breaking change of dropping old node support...

Reply:
can do!

Reply:
https://github.com/emberjs/ember-render-modifiers/pull/28